modules/pubmaticBidAdapter.js

```diff
@@ -537,12 +538,21 @@ function _createBannerRequest(bid) {
   return bannerObj;
 }
 
+export function checkVideoPlacement(videoData) {
+  // Checking if placement property send to video config also
+  // Checking if property is not empty.
+  if (!videoData.hasOwnProperty('placement') || !videoData.placement) {
```

Comment:
you can use utils.deepAccess to retrieve videoData.placement w/o adding checks, the util function will return undefined if the property in the access chain is not present

Reply:
Replaced above code with utils.deepAccess method